lib/start.js

```diff
@@ -1,13 +1,26 @@
 (function() {
   'use strict';
 
-  exports = module.exports = {
-    foo: testMsg
-  }
+  var shell = require('shelljs');
+  var path = require('path');
+  var nconf = require('nconf');
+  var configFile = path.join(process.env.npm_config_quichean_nconf_path,
+    'pytools', 'config.json');
+  var user = process.env.npm_package_config_user;
+  var service = path.join(__dirname, 'service.py');
+  var dbconn = process.env.npm_package_config_db;
+  var args;
+  var venv;
 
-  function testMsg() {
-    console.log('@optbot/template: testing');
-  }
-
-  console.log('@optbot/template: other modules only see this on require()');
+  nconf.file({file: configFile});
```

Comment:
It's not a huge deal, but specifying only "file" is redundant in this case (without more options). There's an overload that just accepts the path as a string.